packages-exp/auth-exp/index.ts

```diff
@@ -25,6 +24,103 @@ import { browserLocalPersistence } from './src/platform_browser/persistence/loca
 import { indexedDBLocalPersistence } from './src/platform_browser/persistence/indexed_db';
 import { browserPopupRedirectResolver } from './src/platform_browser/popup_redirect';
 
+import {
+  // Classes
+  ActionCodeURL,
+  AuthCredential,
+  EmailAuthProvider,
+  MultiFactorResolver,
+  OAuthCredential,
+  PhoneAuthCredential,
```

Comment:
We can't export classes as types because it allows people to new them. Should they be interface instead? If we want to export the actual classes, the implementation should be exported instead.

Reply:
Seems like MultiFactorResolver can be an interface. The other ones are real classes that are already exported, so I am removing those type exports. They could maybe be removed from auth-types-exp/index.d.ts entirely but that's where the documentation comments are and the code class is linking to them using `@inheritDoc`.